README.md

```diff
@@ -1,2 +1,75 @@
-# apm-config
-Semantic-release shareable config for releasing atom packages
+# @semantic-release/apm-config
+
+[Semantic-release](https://github.com/semantic-release/semantic-release) shareable config for releasing atom packages with [apm](https://github.com/atom/apm).
+
+[![Travis](https://img.shields.io/travis/semantic-release/apm-config.svg)](https://travis-ci.org/semantic-release/apm-config)
+[![Codecov](https://img.shields.io/codecov/c/github/semantic-release/apm-config.svg)](https://codecov.io/gh/semantic-release/apm-config)
+[![Greenkeeper badge](https://badges.greenkeeper.io/semantic-release/apm-config.svg)](https://greenkeeper.io/)
+
+## Usage
+
+### Local installation
+
+```bash
+$ npm install --save-dev semantic-release @semantic-release/apm-config
+```
+
+In `package.json`:
+
+```json
+{
+  "release": {
+    "extends": "@semantic-release/apm-config"  
+  }
+}
+```
+
+### Global installation
+
+```bash
+$ npm install -g semantic-release @semantic-release/apm-config
+$ semantic-release -e @semantic-release/apm-config
+```
+
+## Configuration
+
+### Atom installation
+
+The `apm` command line has to be installed in your CI environment and available in the `PATH`.
+
+See the [Atom Package CI Scripts](https://github.com/atom/ci#atom-package-ci-scripts) documentation.
+
+### Github authentication
```

Comment:
*GitHub* should be correctly capitalized everywhere.

Reply:
For some reasons I was convinced that *Github* was the correct capitalization, but you're right it's * GitHub* indeed.

It's fixed now